samples/stock/demo/stock-css-design/demo.js

```diff
@@ -0,0 +1,150 @@
+(async () => {
+
+    // Load the dataset
+    const data = await fetch(
+        'https://demo-live-data.highcharts.com/aapl-ohlcv.json'
+    ).then(response => response.json());
+
+    // split the data set into ohlc and volume
+    const ohlc = [],
+        volume = [],
+        dataLength = data.length;
+
+    for (let i = 0; i < dataLength; i += 1) {
+        ohlc.push([
+            data[i][0], // the date
+            data[i][1], // open
+            data[i][2], // high
+            data[i][3], // low
+            data[i][4] // close
+        ]);
+
+        volume.push([
+            data[i][0], // the date
+            data[i][5],
+            data[i][1] < data[i][4] ? 'highcharts-point-up' : 'highcharts-point-down' // the volume
```

Comment:
If I am not mistaken, we need to compare current close with the previous close to determine the color of volume.

Reply:
In this example, the bar should have the same color as a candle. The way you advice would be a formula for heikenashi :)